### DISCUSSION TITLE
Remove zone from map

### DISCUSSION THREAD

**Discussion by tomlut** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#441] [Q&A] [answered]
I just performed a couple of zoned cleans which worked perfectly. 

How do I now remove these zones from the map?

![Screenshot 2022-07-21 at 10-22-38 Overview – Home Assistant](https://user-images.githubusercontent.com/10679300/180104256-184c3a10-aaf4-4572-b7f8-6e4644ee42ba.png)

**Comment by PiotrMachowski** [comment, marked as answer]
They are drawn on the map image itself and will disappear after start of cleaning.
You can disable them completely in the config of Map Extractor camera by `draw` configuration (basically provide all values except `zones`), but as a result they won't be drawn even during cleaning

**Comment by tomlut** [comment]
Perfect. Thanks for the quick answer.
